src/orb.yml.hbs

```diff
@@ -132,6 +132,31 @@ jobs:
           "The name of the service to update. If undefined, we assume `family` is the name of both the service and task definition."
         type: string
         default: ""
+      deployment-controller:
+        description:
+          "The deployment controller to use for the service. Defaulted to ECS"
+        type: string
+        default: "ECS"
+      application-name:
+        description:
+          "The name of an AWS CodeDeploy application."
+        type: string
+        default: ""
+      deployment-group-name:
```

Comment:
Could this be changed to codedeploy-deployment-group-name for clarity?

Reply:
Fixed by 80b9c36 .